lk2nd/device/dts/msm8909/apq8009w-wtp.dts

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+#include <skeleton64.dtsi>
+#include <lk2nd.dtsi>
+
+/ {
+	model = "Qualcomm APQ8009W";
```

Comment:
Do you need the "model" to boot?

Reply:
Unfortunately, otherwise the device reboots and fastboot returns "No such device"

Reply:
I see, could you then
```suggestion
	model = "Qualcomm APQ8009W"; /* Required by the oem bootloader. */
```

Reply:
I by mistake press shift+enter on the word "Add" but I don't know if this commit message matters or not? If it does I can revert it.

Reply:
You will need to squash all changes into one commit when you're done (and probably drop co-developed tags for review suggestions like this)